php/commands/core.php

```diff
@@ -1150,6 +1164,29 @@ private function get_updates( $assoc_args ) {
 		return array_values( $updates );
 	}
 
+	private function _maybe_remove_unused_files( $version_from, $version_to) {
+		if ( ! $version_from || ! $version_to ) {
+			return;
+		}
+
+		$files_to_remove = array();
+		$includes_folder = defined( 'WPINC' ) ? WPINC : '/wp-includes';
+
+		if ( version_compare( $version_from, '4.4', '>=' ) && version_compare( $version_to, '4.4', '<' ) ) {
```

Comment:
You can use `self::get_core_checksums()` to get a full list of the files present in each version, and then compare the two.